include/Estimation/KrigingSystem.hpp

```diff
@@ -255,6 +256,7 @@ class GSTLEARN_EXPORT KrigingSystem
   MatrixSquareSymmetric _priorCov;  // Dimension NF * NF
   VectorDouble          _postMean;
   MatrixSquareSymmetric _postCov;
+  CholeskyDense         _postCovChol;
```

Comment:
Are we sure that it will always be dense? evalCovMatrix has a sparse version (useful with tapering).